src/module/getData.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable import/no-cycle */
+import cards from './cards.js';
+
+const getData = async () => {
+  const response = await fetch('https://api.tvmaze.com/shows');
+  const getJsonObj = await response.json();
+  const result = getJsonObj.slice(0, 18);
+  const likes = await fetch(
+    'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/5cAzmpr4jeQVeyEjNyKs/likes',
+  );
+  const res = await likes.json();
+  cards(result, res);
+};
+const requestLikes = async (url, id) => {
+  await fetch(url, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      item_id: id,
+    }),
+  });
+  getData();
+};
+requestLikes();
+
+const getComments = async (id) => {
+  const comments = await fetch(`https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/5cAzmpr4jeQVeyEjNyKs/comments?item_id=item${id}`);
```

Comment:
_[OPTIONAL]_ consider saving this URL as a global variable to keep your code DRY as it will mitigate the chances of confusion to its readers

Reply:
```suggestion
  const comments = await fetch(`https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/5cAzmpr4jeQVeyEjNyKs/comments;
  
  const endPoint = '?item_id='
```